bokehjs/src/lib/models/glyphs/webgl/regl_wrap.ts

```diff
@@ -30,8 +30,8 @@ export class ReglWrapper {
   // Drawing functions.
   private _solid_line?: ReglRenderFunction
   private _dashed_line?: ReglRenderFunction
-  private _marker_no_hatch_map?: Map<MarkerType, ReglRenderFunction>
-  private _marker_hatch_map?: Map<MarkerType, ReglRenderFunction>
+  private _marker_no_hatch_map?: Map<MarkerType | "rect", ReglRenderFunction>
+  private _marker_hatch_map?: Map<MarkerType | "rect", ReglRenderFunction>
```

Comment:
Changes like this are generalized in PR #12704.

Reply:
Understood.